util/fibers/detail/scheduler.cc

```diff
@@ -339,6 +378,7 @@ bool Scheduler::WaitUntil(chrono::steady_clock::time_point tp, FiberInterface* m
   return has_timed_out;
 }
 
+// TODO: Optimize
```

Comment:
not clear the intent. either expand or remove

Reply:
?

Reply:
Oh, yes, sorry

Reply:
There's a 100 iteration loop that I'd like to get rid of 🤓